README.md

```diff
@@ -62,11 +57,39 @@ Retro happens at 3pm on Fridays. During retro, we discuss things that happened o
 As with standup, if you are not able to make retro, please let the team know not to wait for you.
 
 ### Check-ins
-Everyone should try and have at least one 1:1 conversation with your manager (Catherine with Stephen, Vyki with Catherine, Stephen with an advisor who will be helpful)
+Everyone should try and have at least one 1:1 conversation with your manager and/or an outside expert adviser each week.
 We love hanging out and coworking in person, but we believe that our team can function effectively remotely.
 Our team members are recruited because of their ability to work independently with integrity and effectiveness. We don’t track hours, we track deliverables. How and when you work s up to you, as long as deliverables are met in a timely manner and any potential issues, questions, or problems are promptly brought to the team’s attention.
 
 ## General dos and don'ts
 Don’t assign “microtasks” to other people. It just makes everyone less productive.
 BAD: “Hey did you book that meeting with Emily?” “No, I don’t have her email. Can you send it to me?”
 GOOD: “Hey did you book that meeting with Emily?” “No, but I’m on it! Where can I find her email?”
+
+## Our communication and management tools
+We use many systems and tools to communicate and work productively - descriptions below are general guidelines.
+Communication:
+Email, Mailchimp - external communication
+Slack, Google Hangouts- internal communication
+Team blog - internal communication/ memorialized updates
+Google group - communication with investors and advisers
+Govtech blog, Medium, and social media accounts: Communication with the community
+
+### File storage
+- We use Google docs as our primary document repository and to collaborate on our short-term working docs, drafts, and personal work.  Store any CityGrows-related documents and work product on the Google Drive - and as we get bigger we’ll probably need an organizing principle for those documents.
+- We use Dropbox as our secondary storage system - anything permanent that isn't expected to change much / doesn't require collaboration, graphic files, etc.
```

Comment:
@Geanuracos what about changing this to "non-document file storage ie images, pdfs, wireframes and mockups, and anything else that Google Drive doesn't support"